### DISCUSSION TITLE
Muting questions

### DISCUSSION THREAD

**Discussion by kerim371** [slimgroup/JUDI.jl#138] [General]
Hi,

In all LSRTM examples we apply muting. I'm trying to understand how the muting works.

In the [documentation](https://slimgroup.github.io/JUDI.jl/dev/inversion/#LSRTM) I've found that this is done _"to suppress the ocean-bottom reflection in the data and the source/receiver imprint in the image"_.

I displayed "marmousi_2D.segy" and "marmousi_migration_velocity.h5" from the mentionned documentation section:
![image](https://user-images.githubusercontent.com/43808863/184850459-fd5665cf-577f-4a5e-a5a0-147aa50a1e90.png)

Few things I can't understand:
1) I think we can't see a direct wave on the seismogram that penetrates in the water from source to receivers. Was it already somehow muted?
2) can't understand do we wan't to mute the direct wave or the region near source position (say a rectangle in 2D experiment)? and if we mute the model then I think we simply set 0 slowness (Inf velocity) to selected region?
3) we have `judiMarineTopmute2D` function but there is no `judiMarineTopmute3D`. Also how to mute 2D/3D land data?
4) if I manually mute the model and SEGY file will it have the same effect as applying JUDI muting at the same regions? The muting is applied to the field data only or to the modeled data as well?
5) if I use 2D filters (based on tau-p or 2D Fourier) instead may it replace muting?
6) the model has spacings 5x5 meters that means that ocean bottom reflection will come at: to=41(points)*5(m)*2(two way travel time)/1.5(m/ьs)=273.3(ms)
But on the seismogram we can see that the reflection is registered at to=136.7(ms) wich is one way travel time. Can we explain that or it is just a misspell of spacings in the model or something that doesn't have much sense?

Would appreciate for any recommendations on that.

P.S. excuse me if my questions are too primitive

**Comment by mloubout** [comment]
There is no primitive questions,  every question is a way to improve the software. Lemme try to answer:

1.  This is an ocean bottom setting. the source is at the surface and the receivers at 210m depth. So the first strong event is the direct wave but it isn't straight due to the dept difference.
2 We want to mute both. First, the direct wave in the data is muted to avoid any artifacts in the image due to its very strong amplitude. Second we want to remove the src/rec imprints in the water layer. These are handled separately via 1judiMarineTopmute` for the data and `judiModeltopmute` for the model
3. The 3D version is indeed not implemented as the muting operation is a bit less trivial. It is on the todo list though. For land data I am not sure as everything is mixed so I suppose you wouldn't apply a muting 
4. You mute both the field and simulated data so that the data misfit is computed properly (direct wave would still be in the simulated data) so doing it in the SEGY wouldn't really help
5. Yes you can use any other operator. We implemented this simple one for our experiments but in practice you would have your own data processing that may be more advanced.
6. That goes back to the first point. This is an OBN setup so it's the one way travel;time for the direct wave

Cheers
